scripts/powershell/Create-Branch.ps1

```diff
@@ -0,0 +1,64 @@
+param (
+    [Parameter(Mandatory=$true)]
+    [ValidatePattern("^[\w\-]+$")]
+    [string]
+    $GitHubRepositoryOwner,
+    
+    [Parameter(Mandatory=$true)]
+    [ValidatePattern("^[\w\-]+$")]
+    [string]
+    $GitHubRepositoryName,
+    
+    [Parameter(Mandatory=$true)]
+    [ValidatePattern("^[\w\-\/]+$")]
+    [string]
+    $GitReference,
+    
+    [Parameter(Mandatory=$true)]
+    [string]
+    [ValidatePattern("^[\w\-\/]+$")]
+    $BranchName,
+    
+    [Parameter(Mandatory=$true)]
+    [string]
+    $GitHubPersonalAccessToken
+)
+
+$ErrorActionPreference = Stop
+
+Write-Host "GitHubRepositoryOwner is: $GitHubRepositoryOwner"
+Write-Host "GitHubRepositoryName is: $GitHubRepositoryName"
+Write-Host "GitReference is: $GitReference"
+Write-Host "BranchName is: $BranchName"
+
+$secureGitHubPersonalAccessToken = ConvertTo-SecureString -String $GitHubPersonalAccessToken -AsPlainText -Force
```

Comment:
Why use SecureString here instead of just passing it in the header like we do in most of our other github calls?